packages/google_sign_in/google_sign_in_platform_interface/lib/google_sign_in_platform_interface.dart

```diff
@@ -89,6 +89,24 @@ abstract class GoogleSignInPlatform {
     throw UnimplementedError('init() has not been implemented.');
   }
```

Comment:
You should update the comment on `init` to say `Deprecated: call initWithParams instead.` instead of `You must call this method before calling other methods.`

Reply:
I also think the statement can be combined to reduce redundancy, something like: "Initializes the plugin with specified `params`".